Monika After Story/game/event-handler.rpy

```diff
@@ -2021,7 +2056,14 @@ label show_prompt_list(sorted_event_keys):
         for event in sorted_event_keys:
             prompt_menu_items.append([unlocked_events[event].prompt,event])
 
-    call screen scrollable_menu(prompt_menu_items, evhand.UNSE_AREA, evhand.UNSE_XALIGN)
+    $ nvm_text = "That's enough for now."
+
+    if madechoice == "unseen":
```

Comment:
`show_prompt_list` isn't used anywhere else, so you don't need to check for the unseen choice here.

Reply:
yeah, i noticed that, i added it in case some time in the future something else does use it. but will remove it.